src/assets/scripts/airport/AirportController.js

```diff
@@ -188,4 +188,21 @@ export default class AirportController {
     hasAirport(icao) {
         return _has(this.airport.airports, icao);
     }
+
+    /**
+     * Remove an aircraft from the queue of any runway(s) at the AirportModel
+     * @for AirportModel
+     * @method removeAircraftFromAllRunwayQueues
+     * @param  {aircraft} aircraft The aircraft to remove
+     */
+    removeAircraftFromAllRunwayQueues(aircraft) {
+        const RUNWAY_PRIMARY_END_INDEX = 0;
+        const RUNWAY_SECONDARY_END_INDEX = 0;
```

Comment:
constants within a method/function can actually be camel case.  It a constant is for an entire file, then they should be uppercase.

are these both `0` index? or is `RUNWAY_SECONDARY_END_INDEX` supposed to be `1`?